source/manual/alerts/search-reindex-failed.html.md

```diff
@@ -0,0 +1,17 @@
+---
+owner_slack: "#2ndline"
```

Comment:
Should we make this `#search-team`, at least to begin with?

Reply:
I don't think we need to. The docs are owned by 2ndline and won't require review for a year...